addon/services/airbrake.js

```diff
@@ -3,7 +3,8 @@ import getClient from '../utils/get-client';
 import setEnvironment from '../filters/environment';
 import loggerReporter from '../reporters/logger';
 import setSession from '../filters/session';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
Currently deprecated. The export will be removed in the next major release of ember-getowner-polyfill.